server/rosetta/converter.go

```diff
@@ -312,7 +312,7 @@ func (c converter) Tx(rawTx tmtypes.Tx, txResult *abci.ResponseDeliverTx) (*rose
 	}
 	// get operations from msgs
 	msgs := tx.GetMsgs()
-
+	//nolint: prealloc
```

Comment:
This github actions failure looks weird to me.. I run `golangci-lint run` locally and it does not exhibit any failures. Any suggestions how I can fix that?